recipes/ocr_pre_annotations.ipynb

```diff
@@ -1,12 +1,23 @@
 {
```

Comment:
<p> </p><p>We can now start to add OCR pre-annotations to the asset metadata:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>